### PR TITLE
Use odo logo instead of author image in blog posts

### LINKED ISSUE #6807: Update odo blog release with odo thumbnails
When we release blog post or documentation links on social media (Twitter, LinkedIn), it should have the odo logo or any new graphic related to the release.

Based on that thumbnail, it displays correctly on the social media site. We need to make sure that image is updated.

### PR DESCRIPTION
**Pull request by valaparthvi**
**What type of PR is this:**
/area documentation
/area release-eng

**What does this PR do / why we need it:**
While sharing blogs on social media platform, the author image is highlighted in big picture which is not always desirable. Using odo logo instead seems like a better option.

**Which issue(s) this PR fixes:**

Fixes #6807 

**PR acceptance criteria:**

- [ ] Unit test 

- [ ] Integration test 

- [ ] Documentation 

**How to test changes / Special notes to the reviewer:**

### CODE REVIEW

File: .github/workflow-data/release-management/blog-post-header.tpl.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 title: odo ${TAG_NAME} Released
 author: $GITHUB_ACTOR
 author_url: https://github.com/$GITHUB_ACTOR
-author_image_url: https://github.com/$GITHUB_ACTOR.png
+author_image_url: https://raw.githubusercontent.com/redhat-developer/odo/main/docs/website/static/img/logo.png
```
Comment (kadel) [inline .github/workflow-data/release-management/blog-post-header.tpl.md:5]: We should probably keep `author_image_url` as it was; it is correct that it points to GitHub avatar. 

According to Docusarus [documentation](https://docusaurus.io/docs/next/api/plugins/@docusaurus/plugin-content-blog) there is also an [`image`](https://docusaurus.io/docs/next/api/plugins/@docusaurus/plugin-content-blog#image) field that should be used to this 

We should probably set `image` to odo logo for this template and all existing blog posts.

Comment (valaparthvi) [inline .github/workflow-data/release-management/blog-post-header.tpl.md:5]:
```suggestion
author_image_url: https://github.com/$GITHUB_ACTOR.png
image: https://raw.githubusercontent.com/redhat-developer/odo/main/docs/website/static/img/logo.png
```

File: .github/workflow-data/release-management/blog-post-header.tpl.md [hunk 2]
```diff
@@ -3,6 +3,7 @@ title: odo ${TAG_NAME} Released
 author: $GITHUB_ACTOR
 author_url: https://github.com/$GITHUB_ACTOR
 author_image_url: https://github.com/$GITHUB_ACTOR.png
+image: https://raw.githubusercontent.com/redhat-developer/odo/main/docs/website/static/img/logo.png
```
Comment (rm3l) [inline .github/workflow-data/release-management/blog-post-header.tpl.md:6]: Can you also add this to all existing release announcement posts?

**Review by rm3l — COMMENTED** [review]
This PR fixes https://github.com/redhat-developer/odo/issues/6807, right?

**Comment by valaparthvi** [comment]
<details><summary>Works as expected!</summary>
<p>


image


</p>
</details>

**Review by rm3l — APPROVED** [review]

**Pull request merged**